src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -70,10 +69,8 @@ public HubConnectionContext(ConnectionContext connectionContext, HubConnectionCo
 
             HubCallerContext = new DefaultHubCallerContext(this);
 
-            if (AppContext.TryGetSwitch("Microsoft.AspNetCore.SignalR.UseAbsoluteClientTimeout", out var useAbsoluteClientTimeout))
-            {
-                _useAbsoluteClientTimeout = useAbsoluteClientTimeout;
-            }
+            _systemClock = _connectionContext.Features.Get<ISystemClock>() ?? new SystemClock();
```

Comment:
Is this to avoid more parameters in the constructor?

Reply:
Yeah